src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/CommonTestTimeouts.cs

```diff
@@ -23,6 +23,14 @@ public static class CommonTestTimeouts
         /// </summary>
         public static readonly TimeSpan WaitForExit = TimeSpan.FromSeconds(15);
 
+        /// <summary>
+        /// Default timeout for gcdump collection.
+        /// </summary>
+        /// <remarks>
+        /// GCDumps can be quite large and take a significant amount of time to transfer.
+        /// </remarks>
+        public static readonly TimeSpan GCDumpTimeout = TimeSpan.FromMinutes(3);
```

Comment:
This was arbitrarily chosen based off `DumpTimeout`; if someone with more intuition has a better value in mind, please feel free to let me know.